modules/http-helper/http_helper_test.go

```diff
@@ -176,5 +184,7 @@ func failRetryHandler(w http.ResponseWriter, r *http.Request) {
 		w.WriteHeader(http.StatusServiceUnavailable)
```

Comment:
I had implicitly meant to read the body here as well but forgot to add a note. Can you do that? Sorry for the extra round trip this caused!

Reply:
Well, now your mentioning it, it is kind of obvious, so no worries for the extra round trip :slightly_smiling_face: Changed it :+1: